source/features/rgh-feature-descriptions.tsx

```diff
@@ -44,23 +48,13 @@ async function init(): Promise<void | false> {
 							}}/>
 					</a>
 				)}
-				<div className={'flex-auto' + (feature.screenshot ? ' ml-3' : '')}>
-					{ /* eslint-disable-next-line react/no-danger */ }
-					<div dangerouslySetInnerHTML={{__html: feature.description}} className="text-bold"/>
-					<div className="no-wrap" data-turbo-frame="repo-content-turbo-frame">
-						<a href={conversationsUrl.href}>Related issues</a>
-						{
-							location.pathname.endsWith('css')
-								? <> • <a href={location.pathname.replace('.css', '.tsx')}>See JavaScript</a></>
-								: undefined
-						}
-					</div>
-				</div>
 			</div>
-		</div>
+		</div>,
 	);
+}
 
-	wrapAll([commitInfoBox, featureInfoBox], <div className="d-lg-flex"/>);
```

Comment:
I tried placing it next to the commit again but they're never wrapping the box, so it looks awkward.

<img width="1024" alt="Screenshot" src="https://user-images.githubusercontent.com/1402241/215478755-cd1c86ea-9b8c-42bc-92b1-59a990a874f3.png">